src/background.js

```diff
@@ -65,10 +66,21 @@ chrome.tabs.onActivated.addListener(async activeInfo => {
   await writeToLocalStorage({ url })
 })
 
-chrome.alarms.create({
-  periodInMinutes: CHECK_INTERVAL_TIMER_MINUTES,
-  when: Date.now() + 1
-})
+export const createChromeAlarm = async checkIntervalTimerMinutes => {
+  await chrome.alarms.clearAll()
```

Comment:
if you are creating an alarm, I don't expect to clear all the other.
Otherwise you should change the function name